entrypoint.sh

```diff
@@ -0,0 +1,5 @@
+#/!bin/bash
+set -m
+electrs&
+websocat -b ws-l:127.0.0.1:1234 tcp:127.0.0.1:50001&
```

Comment:
would honestly prefer to bring back to foreground the electrs process, so we check logs etc..